std-bits/table/src/main/java/org/enso/table/data/column/operation/map/numeric/LongRoundOp.java

```diff
@@ -0,0 +1,71 @@
+package org.enso.table.data.column.operation.map.numeric;
+
+import org.enso.polyglot.common_utils.Core_Math_Utils;
+import org.enso.table.data.column.operation.map.TernaryMapOperation;
+import org.enso.table.data.column.operation.map.MapOperationProblemBuilder;
+import org.enso.table.data.column.storage.numeric.AbstractLongStorage;
+import org.enso.table.data.column.storage.numeric.LongStorage;
+import org.enso.table.data.column.storage.Storage;
+import org.enso.table.error.UnexpectedTypeException;
+import org.graalvm.polyglot.Context;
+
+import java.util.BitSet;
+
+/** An operation rounding integers. */
+public class LongRoundOp extends TernaryMapOperation<Long, AbstractLongStorage> {
+
+    /** Minimum value for the `n` parameter to `roundDouble`. */
```

Comment:
Typo? `roundDouble` is not used here at all.

```suggestion
    /** Minimum value for the `n` parameter to `roundLong`. */
```

Reply:
Done